R/html.R

```diff
@@ -452,12 +452,10 @@ build_404_page = function(html_head, html_toc, html_foot, build, ...) {
   # We create 404 page if it does not exist
   if (file.exists(found <- '_404.md') || file.exists(found <- '_404.Rmd')) {
     rmd_cur = found
-    xfun::Rscript_call(function() {
-      rmarkdown::render(
-        found, rmarkdown::html_fragment(pandoc_args = c('--metadata', 'title=404')),
-        output_file = p404, quiet = TRUE
-      )
-    })
+    xfun::Rscript_call(rmarkdown::render, list(
+      found, rmarkdown::html_fragment(pandoc_args = c('--metadata', 'title=404')),
+      output_file = p404, quiet = TRUE
+    ))
```

Comment:
When I tested it worked as expected. No error regarding `p404` (or `path_404` before that) being not found. That is why I kept it this way. 

but I cannot make it work with a small example so not sure how it worked. 

It is safer to do this - I'll remind that.